jwst/ramp_fitting/utils.py

```diff
@@ -278,6 +278,9 @@ def output_optional(self, model, effintim):
         self.var_p_seg[self.var_p_seg > 0.4 * LARGE_VARIANCE ] = 0.
         self.var_r_seg[self.var_r_seg > 0.4 * LARGE_VARIANCE ] = 0.
 
+        # Tiny 'weights' values correspond to non-existent segments, so set to 0.
+        self.weights[1./self.weights > LARGE_VARIANCE ] = 0.
```

Comment:
In other cases, you compare with `0.4 * LARGE_VARIANCE`, but here you compare with just `LARGE_VARIANCE`.  Is that OK?  See line 462 in ramp_fit.py, for example.